README.md

```diff
@@ -73,16 +99,26 @@ stats_handle_add_tag(api_latency, "units", STATS_UNITS_SECONDS);
 /* in the API service function */
 uint64_t start_ns = get_nanos();
 /* do work */
-stats_set_hist_instscale(api_latency, get_nanos() - start_ns, -9, 1);
+stats_set_hist_intscale(api_latency, get_nanos() - start_ns, -9, 1);
 ```
 
-This will track the latency of every API call into a histogram called `mycoolapp.api.latency` or in tagged for `latency|ST[app:mycoolapp,subsystem:api,units:seconds]`.  The "intscale" variant of histogram insertion allows us to insert nanoseconds into a metric tracking seconds by specifying that it should be scaled by 10^-9.
+This will track the latency of every API call into a histogram called
+`mycoolapp.api.latency` or in tagged form,
+`latency|ST[app:mycoolapp,subsystem:api,units:seconds]`.  The "intscale"
+variant of histogram insertion allows us to insert nanoseconds into a metric
+tracking seconds by specifying that it should be scaled by 10<sup>-9</sup>.
 
 ## Extraction
 
-As a standalone library, libcircmetrics provides a functional writer mechanism to produce a JSON document with all the data in it.
+As a standalone library, libcircmetrics provides a functional writer mechanism
+to produce a JSON document with all the data in it.
 
-The "simple" format is `{ "key": value }` in JSON.  However, due to JSON's horribly poor specification around numbers, it is often useful to know what type the measurement is, so the "not simple" format is: `{ "key": { "_type": "T", "_value": "V" }}` where T is `s` for strings, `i`, `I`, `l`, `L`, or `n` for `int32_t`, `uint32_t`, `int64_t`, `uint64_t`, or `double`, respectively, or `H` for histograms.
+The "simple" format is `{ "key": value }` in JSON.  However, due to JSON's
+horribly poor specification around numbers, it is often useful to know what
+type the measurement is, so the "not simple" format is: `{ "key": { "_type":
+"T", "_value": "V" }}` where T is `s` for strings, `i`, `I`, `l`, `L`, or `n`
```

Comment:
Instead of `T` and `V` maybe we should use the more common `<type>`, and `<value>` to make it clearer.

Reply:
Thanks, did that as a followup to master.